sdks/go/pkg/beam/core/runtime/exec/fullvalue.go

```diff
@@ -35,9 +36,10 @@ type FullValue struct {
 	Elm  interface{} // Element or KV key.
 	Elm2 interface{} // KV value, if not invalid
 
-	Timestamp typex.EventTime
-	Windows   []typex.Window
-	Pane      typex.PaneInfo
+	Timestamp    typex.EventTime
+	Windows      []typex.Window
+	Pane         typex.PaneInfo
+	Continuation sdf.ProcessContinuation
```

Comment:
Genuine (and possibly dumb) question - I'm not totally clear on how our encoding works, but will this cause the whole ProcessContinuation object to get encoded as part of our responses to the runner?

Reply:
I'm not 100% sure, but I don't think so? We could pull the continuation out and set the field in the FullValue back to nil after we retrieve it though.

Reply:
Ok - I'd like to at least verify that before making this change (@lostluck might know offhand)